apps/member-profile/app/shared/components/company-review.tsx

```diff
@@ -31,8 +37,10 @@ type CompanyReviewProps = {
   reviewerId: string;
   reviewerProfilePicture: string | null;
   reviewedAt: string;
+  studentId?: string;
   text: string;
   title: string;
+  workExperiencesId?: string;
```

Comment:
Since `workExperiencesId` is coming from a non-null field (`workExperiences.id`) we shouldn't need to make this optional